src/components/LoanCollections/KivaClassicLoanCarousel.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+	<div class="tw-pt-4">
+		<transition name="kvfade">
+			<div
+				v-if="isLoading"
+				class="spinner"
+			>
+				<kv-loading-spinner />
+			</div>
+		</transition>
+
+		<kv-carousel
+			v-if="augmentedLoanIds.length > 0 && isVisible"
+			class="tw-w-full tw-overflow-visible md:tw-overflow-hidden"
+			:autoplay="false"
+			:embla-options="{
+				loop: false,
+				align: 'start'
+			}"
+			indicator-style="none"
+			:multiple-slides-visible="true"
+			slides-to-scroll="visible"
+			:slide-max-width="singleSlideWidth"
+			@interact-carousel="onInteractCarousel"
+		>
+			<template v-for="(loanId, index) in augmentedLoanIds" #[`slide${index}`]>
+				<!-- Show View more Card -->
+				<router-link
+					v-if="loanId === 3"
+					:key="`view-more-card-${loanId}`"
+					class="tw-flex tw-items-center tw-h-full tw-w-full hover:tw-bg-brand-50 tw-rounded"
+					:to="cleanUrl"
+					v-kv-track-event="[
+						'Homepage',
+						'click-carousel-view-all-category-loans',
+						`${viewAllLoansCategoryTitle}`]"
+				>
+					<div class="tw-w-full tw-text-center">
+						<h3>{{ viewAllLoansCategoryTitle }}</h3>
+					</div>
+				</router-link>
+
+				<!-- show loan card -->
+				<!-- TODO Re-implement card position analytics -->
+				<kiva-classic-basic-loan-card
+					v-else
+					:item-index="index"
+					:key="`loan-${loanId}`"
+					:loan-id="loanId"
+				/>
+			</template>
+		</kv-carousel>
+	</div>
+</template>
+
+<script>
+import KivaClassicBasicLoanCard from '@/components/LoanCards/KivaClassicBasicLoanCard';
+import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
+import KvCarousel from '~/@kiva/kv-components/vue/KvCarousel';
+
+export default {
+	components: {
+		KvCarousel,
+		KvLoadingSpinner,
+		KivaClassicBasicLoanCard,
+	},
+	props: {
```

Comment:
A couple of these aren't used anywhere. Not sure if you'll need them for  future analytics (eg `rowNumber`)

Reply:
Just keeping as placeholders as we need to implement most of them.